nipype/interfaces/matlab.py

```diff
@@ -191,7 +191,7 @@ def _gen_matlab_command(self, argstr, script_lines):
         else:
             prescript.insert(0, "fprintf(1,'Executing code at %s:\\n',datestr(now));")
         for path in paths:
-            prescript.append("addpath('%s');\n" % path)
+            prescript.append("if ~(ismcc || isdeployed), addpath('%s'); end;\n" % path)
```

Comment:
Maybe a comment to explain what's going on here?

```suggestion
            # addpath() is not available after compliation
            # https://www.mathworks.com/help/compiler/ismcc.html
            # https://www.mathworks.com/help/compiler/isdeployed.html
            prescript.append("if ~(ismcc || isdeployed), addpath('%s'); end;\n" % path)
```